### PR TITLE
Fix `go vet` to work with current Grafeas setup

### PR DESCRIPTION
**Pull request by rxbchen**
Fixed issue regarding `go vet` [#310](https://github.com/grafeas/grafeas/issues/310)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -41,7 +41,7 @@ test: go_protos
 	@go test -v ./...
 
 vet: go_protos
-	@go vet -composites=false ./...
```
Comment (vtsao) [inline Makefile]: By default this is true, just remove the flag.

File: go/v1beta1/storage/storage_test.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ import (
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/grafeas/grafeas/go/name"
-	"github.com/grafeas/grafeas/go/v1beta1/api"
```
Comment (vtsao) [inline go/v1beta1/storage/storage_test.go]: Is this an unused import? If so, remove it.

Comment (rxbchen) [reply go/v1beta1/storage/storage_test.go]: I think this is being used in `func doTestStorage()`

File: Makefile (hunk 1 above)
Comment (rxbchen) [reply Makefile]: Will do :)

**Comment by aysylu** [comment]
It looks like all the requested changes by @vtsao have been addressed, but let's wait for him to take another look and explicitly approve. Thanks for addressing this, @rxbchen!

**Comment by vtsao** [comment]
LGTM

**Pull request merged**